### PR TITLE
Improve API docs for US

### PR DESCRIPTION
**Pull request by joshuataylor**

### CODE REVIEW

File: source/includes/_create_orders.md [hunk 1]
```diff
@@ -11,6 +11,12 @@ You can prevent duplicate orders by using an Idempotency Key header with this en
 
 <aside class='warning'>To find out more about <strong>error messages</strong>, be sure to read up on the <a href="#validation-errors">validation errors</a> and <a href="#http-error-codes">HTTP Error codes</a> to help you get better results.</aside>
 
+Your account will require a card for the currency you will be creating orders for.
+
+If you are creating orders in Australia, you will need to have an [AUD Credit Card](https://app.sendle.com/dashboard/billing?currency=AUD) setup, or if you are creating orders in the US you will need to have a [USD Credit Card](https://app.sendle.com/dashboard/billing?currency=AUD) setup in your billing account to create an order in the relevant currency.
```
Comment (gabriel128) [inline source/includes/_create_orders.md]: Typo, in USD CC, should be `[USD Credit Card](https://app.sendle.com/dashboard/billing?currency=USD)`

File: source/includes/_getting_started.md [hunk 2]
```diff
@@ -38,39 +38,41 @@ Any orders created on the sandbox server will be created in the test mode, they
...
+    "error": "payment_required",
+    "error_description": "The account associated with this API key has no method of payment. Please go to your Account Settings in your Sendle Dashboard and add a payment method."
   }
 ```
 
-To create orders with the Sendle API you must make sure you have confirmed your email address. To do this, check your inbox for the email and click the link.
+To use the Sendle API you need to attach a credit card in the currency you are going to be creating orders for to your Sendle account for invoicing. You can do this on the [Billing page](https://app.sendle.com/dashboard/billing) of your Sendle Dashboard.
 
-You must also accept the dangerous goods terms in your [Pickup Settings](https://www.sendle.com/dashboard/pickup_address/edit)
+If you are creating orders in Australia, you will need to have an [AUD Credit Card](https://app.sendle.com/dashboard/billing?currency=AUD) setup, or if you are creating orders in the US you will need to have a [USD Credit Card](https://app.sendle.com/dashboard/billing?currency=AUD) setup in your billing account to create an order in the relevant currency.
```
Comment (gabriel128) [inline source/includes/_getting_started.md]: Same as above, should be `currency=USD` for the US link

File: source/includes/_ping_server.md [hunk 3]
```diff
@@ -66,7 +66,7 @@ When the api receives a request with a `Sendle ID` and an `API Key` the server r
 
 > Response:
 
-```json
+```html
```
Comment (cdbax) [inline source/includes/_ping_server.md]: Is this HTML? Isn't it just plain text?

File: source/includes/_create_orders.md [hunk 4]
```diff
@@ -232,10 +238,10 @@ Each collection within the example booking JSON is described in detail in the se
...
-**postcode** | Postcode of destination location.  It must be a four digit string for a valid location.  If the area is cannot be delivered to, response will be a validation error stating the location is unserviceable.
-**state_name** | Must be the destination location's state or territory.  Valid options include: ACT, NSW, NT, QLD, SA, TAS, VIC, WA.  Long-form (i.e. "Northern Territory") is also accepted.
-**country** <div class="optional">optional</div> | Sendle only works within Australia. If absent, Sendle assumes orders are in Australia.  String value under 255 characters in length.  If included, must read "Australia" or "United States".
+**suburb** | Suburb or town where the parcel is to be delivered.  If Sendle cannot service this location, the response will be a validation error stating that the suburb is not serviceable.  Postcode and suburb must match if within Australia. If they do not match, Sendle will return a set of alternatives to choose from.
+**postcode** | Postcode of destination location. If the area cannot be delivered to, the response will be a validation error stating the location is unserviceable.
```
Comment (cdbax) [inline source/includes/_create_orders.md]: Suggest making this description "Postcode or Zip Code of destination..."

File: source/includes/_ping_server.md (hunk 3 above)
Comment (joshuataylor) [reply source/includes/_ping_server.md]: Returns:
`text/html; charset=utf-8`

**Pull request merged**